ScnSoft-Education/LeetCodeTraining/LongestPalindrome/LongestPalindromeSolution.cs

```diff
@@ -0,0 +1,63 @@
+﻿namespace LeetCodeTraining.LongestPalindrome
+{
+    //https://leetcode.com/problems/longest-palindromic-substring/
+    public static class LongestPalindromeSolution
+    {
+        public static string LongestPalindrome(string s)
+        {
+            string longestPalindromeSubstring = String.Empty;
```

Comment:
This variable is too long, please short it somehow, similar to longestPalSubStr...